docs/RELEASE.md

```diff
@@ -14,8 +14,7 @@ Only `master` branch is used for releases.
     in a tag message.
 
     ```bash
-    export TAG=v1.2.3
-    git tag -a "$TAG" -m "$TAG" && git push origin "$TAG"
+    export TAG=v1.17.7 && git pull && git tag -a "$TAG" -m "$TAG" && git push origin master --tags && stack upload .
```

Comment:
- Can you please keep version as `1.2.3` to make it the same as it is on images?
- Do I need to be somehow logged in to `hackage`  or something to be able to run `stack upload .`

Reply:
you need to log in to hackage with stack, it will remember your credentials after setting it up.

Reply:
Thanks!